src/app/components/home-page/home-page.component.scss

```diff
@@ -0,0 +1,52 @@
+.content {
+  width: 100%;
+}
+
+.img-container {
+
+  display: flex;
+  flex-flow: column;
+  flex: 1 1 auto;
+  align-items: center;
+  justify-content: center;
+}
+img {
```

Comment:
ten `img` powinien się zawierać w img-container. w przypadku gdy dodasz jakąś inną fotkę na tą stronę to bedzie pod tą klasę. szczerze co do animacji: nie jestem przekonany aby animować cokolwiek na stronach.

Reply:
okej